Dockerfile.build

```diff
@@ -0,0 +1,13 @@
+FROM python:3-slim AS build-image
+WORKDIR /code
+COPY . .
+RUN pip install pybuilder
+RUN pip install -r requirements.txt
```

Comment:
I missed this. This is not needed as `pybuilder` should take care of installing dependencies : https://github.com/edgexfoundry/cd-management/blob/e5eaa9cdb46a76a075b6e2b40cd7edc05a26b7e7/build.py#L53-L54

Reply:
I thought so too, but pybuilder was not installing dependencies. I was getting build failures without manually adding the pip install lines in the Dockerfile. I also experience the same behavior when running locally outside of docker.

Reply:
I can try it again. Maybe there was some other issue.